samples/interactive-tutorials/prediction/get_prediction.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START retail_prediction_get_prediction]
+# Call Retail API to get predictions from Recommendation AI using simple request.
+#
+import os
+
+import google.auth
+from google.cloud.retail_v2 import (
+    PredictionServiceClient,
+    PredictRequest,
+    Product,
+    ProductDetail,
+    UserEvent,
+)
+
+project_id = google.auth.default()[1]
```

Comment:
I would call this as `os.environ["GOOGLE_CLOUD_PROJECT"]` not as an item in a list.

Reply:
@loferris We use such format `google.auth.default()[1]` in search samples and in many other places. Are you sure we need to change it?